doc/developers/maintainer.rst

```diff
@@ -119,8 +182,8 @@ Making a release
        $ git checkout master
        $ rm stable
        $ ln -s 0.999 stable
-       $ sed -i "s/latestStable = '.*/latestStable = '0.999';" versionwarning.js
-       $ git commit -m "Update stable to point to 0.999" stable
+       $ sed -i "s/latestStable = '.*/latestStable = '0.999';/g" versionwarning.js
```

Comment:
is g necessary?

Reply:
yeah, my `sed` complained about an unfinished `s/` directive or something w/o it.

Reply:
You're right. It needs the / but not the g